src/solver.cpp

```diff
@@ -330,6 +330,11 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     RegisterWithSolver(
         registry, ++id, ConvAsmImplicitGemmV4R1DynamicWrw{}, miopenConvolutionAlgoImplicitGEMM);
 
+    RegisterWithSolver(registry,
```

Comment:
new solver should be added to the end of the solver registry, so that the existing solver's ID will not be chagned

Reply:
@asroy hi, this is changed